packages/core/src/model/config/default.ts

```diff
@@ -1,29 +1,10 @@
-import { ChainId } from '../../constants'
+import { SUPPORT_NETWORKS } from '../../constants/chainId'
+
 import { FullConfig } from './Config'
 
 export const DEFAULT_CONFIG: FullConfig = {
   pollingInterval: 15000,
-  supportedChains: [
-    ChainId.Mainnet,
-    ChainId.Goerli,
-    ChainId.Kovan,
-    ChainId.Rinkeby,
-    ChainId.Ropsten,
-    ChainId.BSC,
-    ChainId.BSCTestnet,
-    ChainId.xDai,
-    ChainId.Localhost,
-    ChainId.Hardhat,
-    ChainId.Polygon,
-    ChainId.Mumbai,
-    ChainId.Theta,
-    ChainId.ThetaTestnet,
-    ChainId.Harmony,
-    ChainId.Moonriver,
-    ChainId.Palm,
-    ChainId.Fantom,
-    ChainId.Avalanche,
-  ],
+  supportedChains: SUPPORT_NETWORKS.map((network) => network.chainId),
```

Comment:
Let's use here `Chain[]` instead of `number[]`.
It will allow to add custom chains.

Reply:
I choose not to change the default in this PR to scope the impact. 
prefer to do the breaking change in future improvement.

Reply:
👍

Reply:
Let's do it in another PR, and have new name, e.g. `supportedNetworks`